forum_leaderboard/static/js/src/forum_leaderboard_studio.js

```diff
@@ -0,0 +1,21 @@
+/* Javascript for ForumLeaderboardXBlock. */
+function ForumLeaderboardStudioXBlock(runtime, element) {
+    $('.save-button', element).bind('click', function() {
+        var handlerUrl = runtime.handlerUrl(element, 'studio_submit');
+        var data = {
+            discussion_id: $(element).find('input[name=discussion_id]').val(),
+            count: $(element).find('input[name=count]').val()
+        };
+        $.post(handlerUrl, JSON.stringify(data)).success(function(response) {
+            if (response['success']) {
+                window.location.reload(false);
```

Comment:
@Kelketek I'm not sure if it's a common practice to reload an entire page on XBlock edit. Aren't there partial updates?